### PR TITLE
[OUDS] Change `boosted` package to `ouds-web`

### PR DESCRIPTION
**Pull request by julien-deramond**
### Description

The idea of this PR is to move forward with the packaging of OUDS Web, so basically change "boosted" to "ouds-web", and so, all of JS, CSS, and Sass files too.

This is the minimal modifications I can do at first. Not everything is modified correctly, and will be done in separate PRs:
* For now, voluntarily, on the JS side, not everything is modified in the documentation, especially the parts where we have `import * as boosted from 'boosted'`
* Some files are not imported correctly, or "Boosted" is still mentioned somewhere
* Some files are still named "boosted*"

### CODE REVIEW

File: site/content/docs/0.1/getting-started/rtl.md [hunk 1]
```diff
@@ -164,7 +164,7 @@ Need both LTR and RTL on the same page? Thanks to [RTLCSS String Maps](https://r
   } ]
 } */
 .ltr {
-  @import "../node_modules/boosted/scss/boosted";
+  @import "..//node_modules/ouds-web/scss/boosted";
```
Comment (louismaximepiton) [inline site/content/docs/0.1/getting-started/rtl.md:167]:
```suggestion
  @import "../node_modules/ouds-web/scss/ouds-web";
```

**Review by louismaximepiton — COMMENTED** [review]
We're missing some `import "boosted/...` in some `.md` files such as `color-palette.md`, `optimize.md`, `sass.md`, `api.md`, or do we keep them for later?
A `/boosted` is still there in `contribute.md`, `download.md` (8occurrences), `javascript.md` (3), `parcel.md`(6)

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "boosted",
-  "description": "Orange Boosted with Bootstrap is a Bootstrap based, Orange branded accessible and ergonomic components library.",
+  "name": "ouds-web",
+  "description": "OUDS Web is a Bootstrap based, Orange branded accessible and ergonomic components library.",
```
[the remark this answers is not in the document]
Comment (julien-deramond) [reply package.json:3]: Done

**Review by louismaximepiton — APPROVED** [review]
GJ 👌

**Pull request merged**